5.0/en/0x02-Preface.md

```diff
@@ -1,57 +1,29 @@
 # Preface
 
-Welcome to the Application Security Verification Standard (ASVS) version 5.0!
+Welcome to the Application Security Verification Standard (ASVS) Version 5.0.
 
 ## Introduction
 
-Established in 2008 through a collaborative community effort, the ASVS defines the security requirements to be considered when designing, developing, and testing modern web applications and web services.
+Originally launched in 2008 through a global community collaboration, the ASVS defines a comprehensive set of security requirements for designing, developing, and testing modern web applications and services.
 
-ASVS version 5.0 is the culmination of significant effort from the leaders, the working group, and other community members to update and improve this important standard.
+Following the release of ASVS 4.0 in 2019 and its minor update (v4.0.3) in 2021, Version 5.0 represents a significant milestone—modernized to reflect the latest advances in software security.
 
-Our goal for this version has been to make the ASVS easier to use while also making it more clearly focused on its defined scope and covering new, important areas of application development.
+ASVS 5.0 is the result of extensive contributions from project leaders, working group members, and the wider OWASP community to update and improve this important standard.
 
-## Key Objectives from ASVS Version 5.0
+## Principles behind version 5.0
 
-Version 5.0 has been developed with several key principles in mind.
+This major revision has been developed with several key principles in mind:
 
-### Clear Set of Requirements
+* Refined Scope and Focus: The standard now aligns more directly with the foundational pillars in its name: Application, Security, Verification, and Standard. Requirements have been rewritten to emphasize the prevention of security flaws rather than mandating specific technical implementations. Each requirement is now crafted to be self-explanatory, highlighting its rationale.
```

Comment:
4 times "now" in this list

> The standard now aligns more directly

"The standard is made to align more directly"

---

> Each requirement is now crafted to be self-explanatory, highlighting its rationale.

It is a direction, can not state the statement is satisfied.

Reply:
> 4 times "now" in this list

Not sure that is a problem if it is across multiple list items.

> "The standard is made to align more directly"

Maybe:

"This version of the standard has been designed to align more directly..." 

Or just leave it like it is.

> > Each requirement is now crafted to be self-explanatory, highlighting its rationale.

> It is a direction, can not state the statement is satisfied.

Suggest:

"Each requirement is now crafted to be self-explanatory, highlighting its rationale and why it exists."

Reply:
> Not sure that is a problem if it is across multiple list items.

I think the "now" should not be used at all. "now" is valid only _at the moment_, standard lives many years.

---

> This version of the standard has been designed to align more directly

Better than the current version.

---

> Each requirement is now crafted to be self-explanatory, highlighting its rationale and why it exists.

The statement is not valid, and each requirement does not have it. It is a provocative call to report all requirements that do not satisfy the statement. It is our direction, but it is not valid to state for every requirement.

And the "now" is not needed here as well.

Reply:
Ok what do you think of the changes here:
https://github.com/OWASP/ASVS/pull/3129/commits/91a5567b60462dbfa5f4bc786cada4c3abbead65